go/autoscaler/autoscaler.go

```diff
@@ -82,8 +95,8 @@ type Autoscaler struct {
 	eventCh chan event
 }
 
-// NewAutoscaler creates a new Autoscaler.
-func NewAutoscaler(cluster Cluster, options ...func(*Autoscaler)) *Autoscaler {
+// New creates a new Autoscaler.
+func New(cluster Cluster, options ...func(*Autoscaler)) *Autoscaler {
```

Comment:
Rename to `New`, can be referred from outside of the package as `autoscaler.New`, rather than `autoscaler. NewAutoscaler`.